genomic_data_service/region_indexer.py

```diff
@@ -34,11 +35,11 @@
     },
     'dnase-seq': {
         'output_type': ['peaks'],
-        'file_type': ['bed narrowPeak'],
+        'file_type': ['bed narrowpeak'],
         'file_format': ['bed']
     },
     'faire-seq': {
-        'file_type': ['bed narrowPeak'],
+        'file_type': ['bed narrowpeak'],
```

Comment:
I found a few files that were not matching because of string case. I modified the script to always apply `.lower()` before any comparison.